src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -97,156 +185,1886 @@ static DamageTable D_80A164C4 = {
+};
 
+EnGoStruct* func_80A10FD0(EnGoStruct ptr[], Vec3f arg1, Vec3f arg2, Vec3f arg3, f32 arg4, f32 arg5, s32 arg6) {
+    s32 i;
+
+    for (i = 16; i < 32; i++, ptr++) {
+        if (ptr->unk_00 == 0) {
+            ptr->unk_00 = 7;
+            ptr->unk_01 = (Rand_ZeroOne() * (2.0f * (arg6 / 3.0f))) + (arg6 / 3.0f);
+            ptr->unk_02 = ptr->unk_01;
+            ptr->unk_10 = arg1;
+            ptr->unk_1C = arg2;
+            ptr->unk_28 = arg3;
+            ptr->unk_34 = arg4;
+            ptr->unk_38 = arg5;
+            break;
+        }
+    }
+
+    return ptr;
+}
+
+void func_80A11144(EnGoStruct ptr[], GlobalContext* globalCtx) {
+    s32 pad;
+    s32 i;
+    s32 flag = false;
+    f32 temp;
+
+    OPEN_DISPS(globalCtx->state.gfxCtx);
+
+    func_8012C2DC(globalCtx->state.gfxCtx);
+
+    for (i = 0; i < 32; i++, ptr++) {
+
+        if (ptr->unk_00 == 7) {
+            gDPPipeSync(POLY_XLU_DISP++);
+
+            if (!flag) {
+                gSPDisplayList(POLY_XLU_DISP++, object_oF1d_map_DL_0031A0);
+                flag = true;
+            }
+
+            Matrix_StatePush();
+
+            temp = ((f32)ptr->unk_02 / ptr->unk_01);
+            temp *= 255;
+
+            gDPSetPrimColor(POLY_XLU_DISP++, 0, 0, 195, 225, 235, (u8)temp);
+            gSPSegment(POLY_XLU_DISP++, 0x08,
+                       Gfx_TwoTexScroll(globalCtx->state.gfxCtx, 0, (ptr->unk_02 + (i * 3)) * 3,
+                                        (ptr->unk_02 + (i * 3)) * 15, 0x20, 0x40, 1, 0, 0, 0x20, 0x20));
+
+            Matrix_InsertTranslation(ptr->unk_10.x, ptr->unk_10.y, ptr->unk_10.z, MTXMODE_NEW);
+            Matrix_NormalizeXYZ(&globalCtx->billboardMtxF);
+            Matrix_Scale(ptr->unk_34, ptr->unk_34, 1.0f, MTXMODE_APPLY);
+
+            gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx),
+                      G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+            gSPDisplayList(POLY_XLU_DISP++, object_oF1d_map_DL_003258);
+
+            Matrix_StatePop();
+            if (globalCtx->state.gfxCtx) {}
+        }
+    }
+
+    CLOSE_DISPS(globalCtx->state.gfxCtx);
+}
+
+void func_80A1143C(EnGoStruct ptr[], Vec3f arg1, Vec3f arg2, Vec3f arg3, f32 arg4, f32 arg5, s32 arg6, s32 arg7) {
+    s32 i;
+
+    for (i = 16; i < 32; i++, ptr++) {
+        if (ptr->unk_00 == 0) {
+            ptr->unk_00 = arg7 + 4;
+
+            ptr->unk_01 = (Rand_ZeroOne() * (2.0f * (arg6 / 3.0f))) + (arg6 / 3.0f);
+            ptr->unk_02 = ptr->unk_01;
+
+            ptr->unk_10 = arg1;
+            ptr->unk_1C = arg2;
+            ptr->unk_28 = arg3;
+            ptr->unk_34 = arg4;
+            ptr->unk_38 = arg5;
+            break;
+        }
+    }
+}
+
+void func_80A115B4(EnGoStruct ptr[], GlobalContext* globalCtx) {
+    static UNK_TYPE D_80A16644[] = {
+        &D_0408F7E0, &D_0408F3E0, &D_0408EFE0, &D_0408EBE0, &D_0408E7E0, &D_0408E3E0, &D_0408DFE0, &D_0408DBE0,
+    };
```

Comment:
Textures from gameplay keep. Should be able to include there symbols and change type to `TEXTURE_PTR`.